src/interop.rs

```diff
@@ -415,11 +415,12 @@ impl TryFrom<&tex::Entry> for Entry {
         }
 
         if let Some(eprint) = map_res(entry.eprint())? {
-            if map_res(entry.eprint_type().map(|c| c.format_verbatim().to_lowercase()))?
-                .as_deref()
-                == Some("arxiv")
-            {
+            let eprint_type =
+                map_res(entry.eprint_type().map(|c| c.format_verbatim().to_lowercase()))?;
+            if eprint_type == Some("arxiv".to_string()) {
                 item.set_arxiv(eprint);
+            } else if eprint_type == Some("pubmed".to_string()) {
+                item.set_pmid(eprint.replace("http://www.ncbi.nlm.nih.gov/pubmed/", ""));
```

Comment:
I don't think this logic is correct. The reference link you mentioned in the issue (https://ftp.mpi-inf.mpg.de/pub/tex/mirror/ftp.dante.de/pub/tex/macros/latex/contrib/biblatex/doc/biblatex.pdf) shows that `eprint` should contain the identifier directly, not the URL. The URL example was actually using the `url = ...` field and was supposed to exhibit the "bad" case, where the "good" case is when you write `eprint = {IDENTIFIER_HERE}; eprinttype = {pubmed}`. So we don't need to (and shouldn't) replace the URL here, as it shouldn't be there.